src/hotspot/share/gc/shared/ptrQueue.hpp

```diff
@@ -178,7 +178,7 @@ class PtrQueue {
     return byte_offset_of(Derived, _index);
   }
 
-  static ByteSize byte_width_of_index() { return in_ByteSize(sizeof(size_t)); }
+  static constexpr ByteSize byte_width_of_index() { return in_ByteSize(sizeof(size_t)); }
```

Comment:
Do we really have to `constexpr` this for `STATIC_ASSERT`? No objection, just curious.

Reply:
My compiler (gcc 9.3.0) complains with

`[...] g1BarrierSetAssembler_x86.cpp:269:60: error: call to non-'constexpr' function 'static ByteSize PtrQueue::byte_width_of_index()' [...]`

otherwise. Probably because of the called `in_ByteSize()` method.

Reply:
Okay.